turbo.json

```diff
@@ -37,15 +40,17 @@
     "GH_TOKEN",
     "GIT_REPO_URL",
     "GIT_REPO_USER",
+    "LEDGER_INTERNAL_ARGS",
     "MATCH_PASSWORD",
     "MY_APP_BUNDLE_ID",
+    "NO_DEBUG_DEVICE",
     "SENTRY_AUTH_TOKEN",
     "SENTRY_ENVIRONMENT",
     "SENTRY_PROJECT",
     "SHORT_BUNDLE_ID",
-    "SUPPLY_UPLOAD_MAX_RETRIES"
+    "SUPPLY_UPLOAD_MAX_RETRIES",
+    "VERBOSE"
   ],
-  "ui": "tui",
```

Comment:
Did you have any issue with `tui` ? was it to debug properly ?

Reply:
Yes, I didn't liked it and move to regular console display. I will roll back this modification.

Reply:
Thx